common/src/types/token.rs

```diff
@@ -240,6 +321,29 @@ lazy_static! {
         }
         exchange_tickers
     };
+    /// The set of stablecoin quote assets
+    pub static ref STABLECOIN_QUOTES: HashSet<Token> = {
```

Comment:
I think we should remove this, it isn't used anywhere. A couple other points on this:
1. We don't need a hashset for two elements, it's almost certainly slower than a vector
2. I don't think we need to lazily initialize anything about our quote tokens, especially given that there are two of them. `Token::from_ticker` is totally fine to do at runtime, and I'd far prefer that to more `lazy_static`s.

Reply:
fair, I'll just do a constant array of quote tickers. but this will be used in the next PR! got a bit ahead of myself in this one and decided to cut it short